tmpl/method.tmpl

```diff
@@ -17,8 +17,8 @@ var self = this;
 <div class="summary">
     <?js if (data.summary) { ?>
     <?js= summary ?>
-    <?js } else { ?>
-    ...
+    <?js } else if (data.description){ ?>
+    <?js= data.description.split('<\/p>')[0]?></p>
```

Comment:
The commit summary says "first line" but you're actually doing "first paragraph".  The javadoc convention was always "first sentence", and I seem to recall that there is already some code in jsdoc which does that, maybe in its core.  It would be good to reuse a standard function here rather than rolling our own, especially because for i18n the "first sentence" split should actually come from libicu or some such.